params/config.go

```diff
@@ -357,6 +357,9 @@ type ChainConfig struct {
 	// Various consensus engines
 	Ethash *EthashConfig `json:"ethash,omitempty"`
 	Clique *CliqueConfig `json:"clique,omitempty"`
+
+	// Optimism config
+	Optimism *OptimismConfig `json:"optimismfee,omitempty"`
```

Comment:
Perhaps the json tag should be `optimism` here

Reply:
Fixed.